precompiles/outposts/osmosis/tx_test.go

```diff
@@ -389,19 +362,19 @@ func (s *PrecompileTestSuite) TestSwap() {
 			sender: senderAddress,
 			origin: senderAddress,
 			malleate: func() []interface{} {
-				_, err := testutils.RegisterEvmosERC20Coins(*s.unitNetwork, sender)
-				s.Require().NoError(err, "expected no error during evmos erc20 registration")
-
 				osmoIbcDenomTrace := utils.ComputeIBCDenomTrace(PortID, ChannelID, osmosis.OsmosisDenom)
 				osmoTokenPair, err := testutils.RegisterIBCERC20Coins(s.unitNetwork, sender, osmoIbcDenomTrace)
 				s.Require().NoError(err, "expected no error during ibc erc20 registration")
 
+				err = evmosutils.FundAccountWithBaseDenom(s.unitNetwork.GetContext(), s.unitNetwork.App.BankKeeper, sender, 1000000000000000000)
+				s.Require().NoError(err, "failed to fund account")
```

Comment:
why is this necessary? We should rather set up the test suite with enough funds or adjust the test case to make use of the existing balance I think - also not sure where this amount comes from, why exactly 1000000000000000000?